### PR TITLE
[ECR-2920] LC: readme + changelog

### PR DESCRIPTION
**Pull request by bullet-tooth**
## Overview

---
See: https://jira.bf.local/browse/ECR-2920

### Definition of Done

- [ ] There are no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [ ] Public API has Javadoc
- [ ] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: exonum-light-client/README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ The following table shows versions compatibility:
 | Light Client | Exonum | Exonum Java |
 |--------------|--------|-------------|
 | 0.1.0        | 0.10.* | 0.4         |
+_Be noticed, that the table above includes release versions only
+and does not include snapshot versions_
```
Comment (dmitry-timofeev) [inline exonum-light-client/README.md]: I'd leave this out as snapshots are internal things. Or are you trying to say "Light client is tested against release versions, compatibility with unreleased versions is not guaranteed"?

**Comment by dmitry-timofeev** [comment]
@Maria-Nosyk, we haven't decided on the content yet, therefore no review is needed until that :)

File: exonum-light-client/README.md [hunk 2]
```diff
@@ -92,6 +94,57 @@ Also, you can take a look at the [integration test][send-tx-it]
...
+* `count` is a number of blocks to return.
+* `skipEmpty` is a parameter to filter empty blocks i.e. blocks without transactions.
+* `maxHeight` is a number to filter blocks starting from the given value in descending order.
+* `showTimes` is a parameter indicates adding block acceptance times to the response.
+
+### Number of unconfirmed transactions
+<!-- TODO: remove after release --> 
+_*Not available for Light Client v0.1_  
+To get a number of currently unconfirmed transactions i.e.
+are waiting to be committed to the blockchain:
```
Comment (MakarovS) [inline exonum-light-client/README.md]: `transactions, i.e., that are waiting`?

File: exonum-light-client/README.md [hunk 3]
```diff
@@ -92,6 +94,57 @@ Also, you can take a look at the [integration test][send-tx-it]
...
+_*Not available for Light Client v0.1_  
+The following method provides a possibility to get an array of blocks
+by the specified range:
+```java
+BlocksResponse response = exonumClient.getBlocks(count, skipEmpty, maxHeight, showTimes);
+```
+* `count` is a number of blocks to return.
+* `skipEmpty` is a parameter to filter empty blocks i.e. blocks without transactions.
+* `maxHeight` is a number to filter blocks starting from the given value in descending order.
+* `showTimes` is a parameter indicates adding block acceptance times to the response.
```
Comment (MakarovS) [inline exonum-light-client/README.md]: `that indicates`?

File: exonum-light-client/README.md [hunk 4]
```diff
@@ -35,6 +46,8 @@ compile 'com.exonum.client:exonum-light-client:0.1.0'
 ```
 
 ## Examples
+Please be noticed that examples below contain most frequently used operations only.
```
Comment (dmitry-timofeev) [inline exonum-light-client/README.md]: I think this notice is redundant.

File: exonum-light-client/README.md [hunk 5]
```diff
@@ -9,14 +9,25 @@ Also, Exonum light client provides access to [common utils][ejb-common]
 toolkit which contains some helpful functions for _hashing_,
 _cryptography_, _serialization_ etc. 
 
+## Capabilities
+By using the client you are able to perform next operations:
```
Comment (dmitry-timofeev) [inline exonum-light-client/README.md]: the following operations

File: exonum-light-client/README.md [hunk 6]
```diff
@@ -9,14 +9,25 @@ Also, Exonum light client provides access to [common utils][ejb-common]
...
+- Submit transactions to the node
+- Receive transaction information 
+- Receive blockchain blocks information 
+- Receive node system information
```
Comment (dmitry-timofeev) [inline exonum-light-client/README.md]: What is that and how is it different from the node status information below?

Comment (bullet-tooth) [reply exonum-light-client/README.md]: os, exonum versions vs health check info

File: exonum-light-client/README.md [hunk 7]
```diff
@@ -92,6 +104,17 @@ Also, you can take a look at the [integration test][send-tx-it]
 for the full example of how to create a transaction message and
 send it to Exonum node.
 
+### Transaction info
+<!-- TODO: remove after release --> 
+_*Not available for Light Client v0.1_  
+The following method provides a possibility to get a transaction
+information, either committed or uncommitted, by the hash
+of the transaction:
```
Comment (Maria-Nosyk) [inline exonum-light-client/README.md]: I would rewrite this as follows:
> The following method provides a possibility to get information on a transaction by its hash - status of the transaction, details of the message containing the transaction and, if available, transaction location and result:

**Pull request merged**